### PR TITLE
PHP 8 Compatibility fixes

### PR DESCRIPTION
**Pull request by markbiek**
## Proposed changes:

These are minor changes as part of our PHP 8.1 compatibility update initiative pdqkMK-UN-p2.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

* pdqkMK-UN-p2
* #23695 

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* These are minor enough that I think visual inspection should be sufficient but please let me know if you need more information.

### CODE REVIEW

File: projects/packages/stats/src/class-tracking-pixel.php [hunk 1]
```diff
@@ -242,7 +242,7 @@ private static function stats_array_to_string( $kvs ) {
 		foreach ( $kvs as $k => $v ) {
 			$jskvs[] = "$k:'$v'";
 		}
-		return join( ',', $jskvs );
+		return implode( ',', $jskvs );
```
Comment (jeherve) [inline projects/packages/stats/src/class-tracking-pixel.php:245]: I'm curious about this. Is `join` discouraged as of PHP 8? I'm not against that change, I'd much rather we use `implode` everywhere, but I don't understand how that's linked to PHP 8? Is `join` going to be deprecated maybe?

Comment (markbiek) [reply projects/packages/stats/src/class-tracking-pixel.php:245]: I think `join` to `implode` is less a PHP 8-specific thing and more of a "standardize across the codebase" thing. I'm not sure who decided on these but there are a few similar substitutions that you can see in https://github.com/Automattic/wpcom-php-migration/blob/main/rector.php#L57

Comment (anomiex) [reply projects/packages/stats/src/class-tracking-pixel.php:245]: Chances are if we really want to care about this sort of thing we should configure phpcs in the monorepo to make these replacements too.

**Review by anomiex — APPROVED** [review]

**Comment by markbiek** [comment]
@anomiex Should I merge this or do you have a process where someone else will merge it?

**Comment by anomiex** [comment]
You can go ahead.

File: projects/packages/stats/src/class-tracking-pixel.php (hunk 1 above)
Comment (markbiek) [reply projects/packages/stats/src/class-tracking-pixel.php:245]: Working on the phpcs change here: https://github.com/Automattic/jetpack/pull/30601

**Pull request merged**